packages/teleport/src/Users/UserList/UserList.tsx

```diff
@@ -40,6 +40,12 @@ export default function UserList({
           key: 'roles',
           headerText: 'Roles',
           isSortable: true,
+          onSort: (a: string[], b: string[]) => {
```

Comment:
Interesting approach to sort based on a list of roles :)

Reply:
i reverted back to basic sorting, so we use basic sorting when we sort the list of roles